docker/variables.env.example.docker

```diff
@@ -0,0 +1,5 @@
+MSSQL_PID = Express
```

Comment:
The proposed `.env.example.docker` file should be in the `EdFi.AnalyticsMiddleTier.Tests` project, rather than here, and it should have the appropriate environment variable names for that project.